packages/react-charting/src/components/HeatMapChart/HeatMapChart.base.tsx

```diff
@@ -294,45 +298,81 @@ export class HeatMapChartBase extends React.Component<IHeatMapChartProps, IHeatM
      * yAxisDataPoint is noting but the DataPoint
      * which will be rendered on the y-axis
      */
-    yAxisDataPoints.forEach((yAxisDataPoint: string, index1: number) => {
-      /**
-       * dataPointObject is an object where it contains information on single
-       * data point such as x, y , value, rectText property of the rectangle
-       */
-      this._dataSet[yAxisDataPoint].forEach((dataPointObject: FlattenData, index2: number) => {
-        const id = `x${index1}y${index2}`;
-        const rectElement: JSX.Element = (
-          <g
-            key={id}
-            role="img"
-            aria-label={this._getAriaLabel(dataPointObject)}
-            data-is-focusable={this._legendHighlighted(dataPointObject.legend) || this._noLegendHighlighted()}
-            fillOpacity={this._getOpacity(dataPointObject.legend)}
-            transform={`translate(${this._xAxisScale(dataPointObject.x)}, ${this._yAxisScale(dataPointObject.y)})`}
-            ref={(gElement: SVGGElement) => {
-              this._rectRefCallback(gElement, id, dataPointObject);
-            }}
-            onFocus={this._onRectFocus.bind(this, id, dataPointObject)}
-            onBlur={this._onRectBlurOrMouseOut}
-            onMouseOver={this._onRectMouseOver.bind(this, id, dataPointObject)}
-            onMouseOut={this._onRectBlurOrMouseOut}
-          >
-            <rect
-              fill={this._colorScale(dataPointObject.value)}
-              width={this._xAxisScale.bandwidth()}
-              height={this._yAxisScale.bandwidth()}
-              onClick={dataPointObject.onClick}
-            />
-            <text
-              dominantBaseline={'middle'}
-              textAnchor={'middle'}
-              className={this._classNames.text}
-              transform={`translate(${this._xAxisScale.bandwidth() / 2}, ${this._yAxisScale.bandwidth() / 2})`}
+    yAxisDataPoints.forEach((yAxisDataPoint: string) => {
```

Comment:
the snapshots should get updated with this change

Reply:
Most of the snapshot tests are behind the TEST env flag. But I see 2 RTL snapshot tests enabled, and I have no idea why these snapshots are not updating.